### DISCUSSION TITLE
[@types/qs] ParsedQs is wrong type

### DISCUSSION THREAD

**Discussion by szulcus** [DefinitelyTyped/DefinitelyTyped#66472] [Issues with a @types package]
`parse` function with `strictNullHandling` option can return `null` value
![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/48328739/6b3a9116-78f4-48a2-9376-f60f826038b8)

[a comment by an automated account is not shown]

**Comment by ljharb** [comment]
Good catch. It would be ideal to pivot on the type of `options` - so that `null` is only an allowed return type when `strictNullHandling` is true. I'd be happy to review that PR.
